File: sdk/python/luml/experiments/backends/_exceptions.py
class LumlExperimentsError(Exception):
    def __init__(
        self,
        message: str = "Luml Experiments error",
    ) -> None:
        self.message = message
        super().__init__(self.message)


class LumlFilterError(LumlExperimentsError, ValueError):
    def __init__(
        self,
        message: str = "Invalid filter string",
    ) -> None:
        super().__init__(message)

File: extras/js/packages/experiments/src/utils/_exceptions.ts
export class LumlExperimentsError extends Error {
  constructor(message = 'Luml Experiments error') {
    super(message);
    this.name = 'LumlExperimentsError';
  }
}

export class LumlFilterError extends LumlExperimentsError {
  constructor(message = 'Invalid filter string') {
    super(message);
    this.name = 'LumlFilterError';
  }
}